R/classes.R

```diff
@@ -55,9 +53,11 @@ print.PartInv <- function(x, ...) {
   cat("Cutpoint on the latent scale (xi): ", round(x$cutpt_xi, 3), "\n")
   cat("Cutpoint on the observed scale (Z): ", round(x$cutpt_z, 3), "\n")
   cat("AI ratio: ", round(x$ai_ratio, 3), "\n\n")
-  print.PartInvSummary(x$summary)
-  if (!is.null(x$plot)) {
-    print(x$plot)
+  cat("Partial invariance results:\n\n")
+  print_PartInvSummary(x$summary)
+  if (!is.null(x$summary_mi)) {
+    cat("\n\nStrict invariance results:\n\n")
+    print_PartInvSummary(x$summary_mi)
   }
```

Comment:
Currently only the summary table for strict invariance is printed when `show_mi=TRUE`,  and the cutpoints and propsel under strict invariance are not reported. How about adding the following to keep consistent with the partial invariance case?

```suggestion
  if (!is.null(x$summary_mi)) {
    cat("\n\nStrict invariance results:\n\n")
    cat("Proportion selected: ", round(x$propsel_mi, 3), "\n")
    cat("Cutpoint on the latent scale (xi): ", round(x$cutpt_xi_mi, 3), "\n")
    cat("Cutpoint on the observed scale (Z): ", round(x$cutpt_z_mi, 3), "\n")
    print_PartInvSummary(x$summary_mi)
  }
```